code/modules/mob/living/carbon/human/human_hud.dm

```diff
@@ -3,15 +3,15 @@
 	if(!H.client)//no client, no HUD
 		return
 
-//	var/datum/hud/human/HUDdatum = global.HUDdatums[H.defaultHUD]
+//	var/datum/hud/human/HUDdatum = GLOB.HUDdatums[H.defaultHUD]
 	var/recreate_flag = FALSE
 
 	if(!check_HUDdatum())//check client prefs
 		log_debug("[H] try check a HUD, but HUDdatums not have \"[H.client.prefs.UI_style]!\"")
 		to_chat(H, "Some problem hase accure, use default HUD type")
 		H.defaultHUD = "ErisStyle"
 		recreate_flag = TRUE
-	else if (H.client.prefs.UI_style != H.defaultHUD)//���� ����� � ���� �� ��������� �� ������ � �������
+	else if (H.client.prefs.UI_style != H.defaultHUD)//���� ����� � ���� �� ��������� �� ������ � �������
```

Comment:
Not sure what happened here?

Reply:
fix eeeet

Reply:
Can i just delete all the comments, mr.headdev?

Reply:
I would yeah. Or replace them with an English counterpart.

Reply:
Could not find English counterparts for "Âëèâàåì â ñîîòâåñòâóþùèé ñïèñîê", "Èùåì òàêîé â äàòóìå", "åñëè ó êëèåíòà ìîáà ïðîïèñàí ñòèëü\òèï ÕÓÄà" and other comments so they have been deleted.